pkg/extension/opampextension/testdata/opamp.d/opamp-rabbitmq-config.yaml

```diff
@@ -0,0 +1,15 @@
+receivers:
+  rabbitmq:
+    endpoint: http://localhost:15672
```

Comment:
is ```http://``` required for rabbitmq?

Reply:
whoops, nvm found it in the readme. that's so weird 😆

Reply:
No its optional - https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/rabbitmqreceiver/README.md

Reply:
the endpoint field is optional. But the scheme needs to be specified